### PR TITLE
Some fixes to first four chapters of Boxes and Diamonds.

### PR DESCRIPTION
**Pull request by ionathanch**

### CODE REVIEW

File: content/normal-modal-logic/completeness/modalities-ccs.tex [hunk 1]
```diff
@@ -178,15 +178,15 @@
...
   only if $\Diamond\Delta \subseteq \Gamma$.
 \end{lem}
 
 \begin{proof}
   ``Only if'' direction: Assume $\Box^{-1}\Gamma \subseteq \Delta$ and
   suppose $\Diamond!A \in \Diamond\Delta$ (i.e., $!A \in \Delta$).  In
-  order to show $\Diamond!A \in \Gamma$ it suffices to show
-  $\Box\lnot!A \notin \Gamma$ for then by maximality $\lnot\Box\lnot
+  order to show $\Diamond!A \in \Gamma$, it suffices to show
+  $\Box\lnot!A \notin \Gamma$, for then by maximality, $\lnot\Box\lnot
```
Comment (ionathanch) [inline content/normal-modal-logic/completeness/modalities-ccs.tex:189]: I found the wording of this sentence to be confusing, especially the "for then by maximality" part, but I wasn't sure how to rearrange it so I just split it up with commas.

File: content/normal-modal-logic/syntax-and-semantics/relational-models.tex [hunk 2]
```diff
@@ -33,7 +33,7 @@
 represented by means of simple diagrams, such as the one in
 \olref{fig:simple}. Worlds are represented by nodes, and world~$w'$ is
 accessible from~$w$ precisely when there is an arrow from~$w$
-to~$w'$. Moreover, we label a node (world) by~$\mTrue{p}$ when $w \in V(p)$,
+to~$w'$. Moreover, we label a node (world) by~$p$ when $w \in V(p)$,
```
Comment (ionathanch) [inline content/normal-modal-logic/syntax-and-semantics/relational-models.tex:36]: Not an error, but `\mTrue` makes a gap in the front for a phantom `\neg`, and within the text this shows as a weird gap.

**Comment by ionathanch** [comment]
I also noticed that [open-logic-envs.sty](https://github.com/OpenLogicProject/OpenLogic/blob/master/open-logic-envs.sty) and [sty/open-logic-book-envs.sty](https://github.com/OpenLogicProject/OpenLogic/blob/master/sty/open-logic-book-envs.sty) pluralize Lemma differently as Lemmas and Lemmata, I'm not sure if that's intentional or if it matters at all. (Personally I think if Lemmata is used then Formulae should be used as well.)

**Comment by ionathanch** [comment]
One last thing, I didn't see "maximality" defined anywhere. Maybe it should be defined in the same Definition as for complete Σ-consistency?

File: content/normal-modal-logic/syntax-and-semantics/relational-models.tex (hunk 2 above)
Comment (rzach) [reply content/normal-modal-logic/syntax-and-semantics/relational-models.tex]: I'll fix it in a different way. Someone might decide to label things in some other way (eg, \models p, or +p, or whatever)

**Comment by rzach** [comment]
Thanks!

**Pull request merged**